site/src/pages/release-notes/1.21.0.mdx

```diff
@@ -0,0 +1,148 @@
+---
+date: 2022-12-20
+---
+
+## 🌟 New features
+
+- You can now use <type://@RequestTimeout> to set a request timeout to annotated services and
+  gRPC services. #4463 #4499
+  ```java
+  import com.linecorp.armeria.server.annotation.decorator.RequestTimeout;
+  
+  @RequestTimeout(value = 1, unit = TimeUnit.SECONDS) // 👈👈👈
+  class MyAnnotationService {
+    // A request gets timed out in 1 second.
+    @Get("/fast/api")
+    public CompletableFuture<MyResponse> fastApi() {
+      ...
+    }
+
+    // A request gets timed out in 5 second.
+    @RequestTimeout(value = 5, unit = TimeUnit.SECONDS) // 👈👈👈
+    public CompletableFuture<MyResponse> slowApi() {
+      ...
+    }
+  }
+  ```
+- You can now set the default `MeterRegistry` by implementing a custom <type://FlagsProvider>. #4541
+  ```java
+  public class CustomFlagsProvider implements FlagsProvider {
+    @Override
+    public MeterRegistry meterRegistry() {
+      return PrometheusMeterRegistries.newRegistry(); // 👈👈👈
+    }
+  }
+  ```
+- You can now set the default <type://HttpHeaders> to <type://Server>, <type://VirtualHost> and
+  <type://Service>. #4520
+  ```java
+  static AsciiString CUSTOM_HEADER =
+    HttpHeaderNames.of("x-custom-header");
+
+  Server
+    .builder()
+    .setHeader(CUSTOM_HEADER, "server-default-header")
+    .virtualHost("foo.com")
+    .setHeader(CUSTOM_HEADER, "vhost-default-header").and()
+    .route()
+    .path("/foo")
+    .setHeader(CUSTOM_HEADER, "service-default-header")
+  ```
+- You can now set <type://Endpoint#host()> and `:authority` or `Host` header
```

Comment:
How about
```
... now set  :authority or Host header whose value is different from <type://Endpoint#host()> using ...
- This will be useful when you want to send a request through proxy. For example:
```